docs/self-managed/installation-methods/helm/configure/database/elasticsearch/using-existing-elasticsearch.md

```diff
@@ -101,7 +126,29 @@ elasticsearch:
   enabled: false
 ```
 
-## Next steps
+### Prerequisites
```

Comment:
I just adjusted the template - Lets move it before the configuration section? (makes more sense to clarify first what is needed then configuring it , or?)